src/components/Footer/Footer.js

```diff
@@ -1,8 +1,64 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './Footer.scss';
+import FooterBox from './FooterBox';
+import FooterIcon from './FooterIcon';
 
 const Footer = () => {
-  return <footer></footer>;
+  const [footerValue, setFooterValue] = useState([]);
+  const [iconValue, setIconValue] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/footerData.json')
+      .then(res => res.json())
+      .then(data => {
+        setFooterValue(data);
+      });
+  }, []);
+  useEffect(() => {
+    fetch('/data/iconData.json')
+      .then(res => res.json())
+      .then(data => {
+        setIconValue(data);
+        console.log(data[0].icon_name);
+      });
+  }, []);
+
+  return (
+    <footer className="foo">
```

Comment:
```suggestion
    <footer className="footer">
```
최상단 element의 className은 컴포넌트 이름과 동일하게 네이밍 해주시는게 위코드 컨벤션입니다.
nesting 할때 중복 최소화 하기위한 하나의 장치 입니다.